apify-docs-theme/src/theme/SearchBar/index.js

```diff
@@ -6,6 +7,9 @@ import React, { useCallback } from 'react';
 
 import { ApifySearch } from '@apify/docs-search-modal';
 
+// needs to be imported as the last thing, so that it can override the default styles
```

Comment:
Nit to add:

`TODO: update simple-import-sort to allow importing css as last.`